tools/apoc/apoc.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="apoc" name="APoc" version="1.0">
+    <description>Large-scale identification of similar protein pockets</description>
+    <requirements>
+        <requirement type="package" version="1b16">apoc</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+        <![CDATA[ 
+        #if '$templates_source.template_source_select'=="list"#
```

Comment:
You shouldn't use quotes around variables in the Cheetah directives (lines starting with `#`), only in the lines that end up in the job script (and for variables corresponding to `<param>s` of type `text`, `data`, `data_collection` or similar).